Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            int prev = equation.IndexOf('*');
+            string A = equation.Substring(0,prev);
+            int next = equation.IndexOf('=');
+            string B = equation.Substring(prev+1,next-prev-1);
+            string C = equation.Substring(next+1);
+            //Console.WriteLine($"{A} {B} {C}");
+            if(C.Contains('?'))
+            {
+                float a = float.Parse(A);

```

Comment:
what is float a? how it is different from A?

Can give some meaningful name to it.